### ISSUE TITLE
number of threads specification

### ISSUE THREAD

**Issue by Aron-github** [denglab/SeqSero#5]
The default value for the number of thread used by SeqSero is 1 and the only way of modifiyng it is to manually alter the code of run_auto_All_for_web_multi_revise.py, run_auto_All_for_assemblies.py, plus the codes using blast/bwa...am I correct? 
Do you think you will include a change in SeqSero 1.0 to account for this as I see you already specified this in the alpha test version of SeqSero 2?

Thanks,
Nora

**Comment by ShaokangZhang** [comment]
Yes, seqsero1 is set to apply 1 thread each time. Yes, we will add the multiple threads module in SeqSero2 for users' convenience.
Besides manually altering codes, if you want to use multiple threads, a much easier way is to use "parallel" of linux. 
For example, 
parallel -j 20 'SeqSero.py -m 1 -i {1}' ::: *.sra
it will apply 20 threads to serotype all sra files.
SeqSero Team
